td-shim-tools/src/enroller.rs

```diff
@@ -161,6 +161,15 @@ pub fn enroll_files(
     Ok(())
 }
 
+// Uncompressed ecdsa p384 public key length
+const ECDSA_P384_PUB_KEY_LEN: usize = 97;
```

Comment:
Why it is 97?
The pub key (x + y) should be 48 + 48 = 96

Reply:
The extra byte is used to mark whether the key is compressed or not

Reply:
I dont see there is such definition in https://github.com/confidential-containers/td-shim/blob/main/doc/secure_boot.md.

Reply:
This is very confusing. a P384 pub key size should be 0x96.
If you add other byte, then the meaning is not P384 pub key size.